.lintstagedrc

```diff
@@ -1,8 +1,8 @@
 {
 	"*.ts": [
-		"balena-lint --fix"
+		"./bin/balena-lint --fix"
 	],
 	"test/**/*.ts": [
-		"balena-lint --tests"
+		"./bin/balena-lint --tests"
```

Comment:
Can you see if `npx balena-lint` works? If so that's be better because it'd avoid needing to rely on the #! of a file to invoke node (which doesn't work so well on windows :P )

Reply:
Good point regarding Windows. Now adopting the following alternative that will be cross platform:

```suggestion
		"node bin/balena-lint --tests"
```

`package.json` already contains a few scripts starting with "node".

`npx balena-lint` does not work:
```
$ npx balena-lint --help
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/balena-lint - Not found
npm ERR! 404
npm ERR! 404  'balena-lint@latest' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/paulo/.npm/_logs/2021-11-25T17_57_07_132Z-debug.log
Install for [ 'balena-lint@latest' ] failed with code 1
```

By the way, note that this change only applies to the development of `node-balena-lint` itself, not projects using `node-balena-lint`.